launch/launch_utils/launch_base.py

```diff
@@ -17,42 +20,37 @@ def get_files():
 
 def get_base_nodes():
     files = get_files()
-    atos_conf_dir = os.path.join(os.path.expanduser('~'), '.astazero', 'ATOS')
     atos_install_dir = get_package_prefix('atos')
 
-    # control-gui logging
-    control_gui_log = open(atos_conf_dir / Path("webgui.log"), 'w')
     # start control-gui server
     control_gui_dir = Path(atos_install_dir) / Path("controlpanel/")
-    subprocess.Popen("npm start --prefix " + str(control_gui_dir),shell=True, stdout=control_gui_log, stderr=control_gui_log)
+
+    insecure_websockets = LaunchConfiguration('insecure')
+    foxbridge = LaunchConfiguration('foxbridge')
+
+    insecure_launch_arg = DeclareLaunchArgument('insecure', default_value='False')
+    foxbridge_launch_arg = DeclareLaunchArgument('foxbridge', default_value='True')
 
     return [
+        foxbridge_launch_arg,
+        insecure_launch_arg,
+        ExecuteProcess(
+            name='control_gui',
+            output={'both': 'log'}, #print to log to avoid cluttering the terminal
+            cmd=[[
+                FindExecutable(name='npm'),
+                ' start --prefix ',
```

Comment:
It would be nice to to see stderr in the terminal to know if node crashes for any reason.

Reply:
Using the currently implemented method, the user is made aware of crashes 
![MicrosoftTeams-image (13)](https://github.com/RI-SE/ATOS/assets/97448034/b9601530-39bd-4782-b1e9-5e874e479c87)